pkg/skaffold/util/clock.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import "time"
+
+//a simple mockable Clock interface
+type Clock interface {
+	Now() time.Time
+}
+
+//a real implementation of Clock
+type RealClock struct{}
+
+func (RealClock) Now() time.Time {
+	return time.Now()
```

Comment:
Or you could just get rid of this. I don't think we'd have other implementations of the interface or other functions on it, so i think it would be safe to just make it

```golang
type dateTimeTagger struct {
	Format   string
	TimeZone string
	timeFn    func()time.Time
}

tagger := dateTimeTagger{timeFn: time.Now}
mockTagger := dateTimeTagger{timeFn: func()time.Time { return test.time } }
```
Might save a bit of code, but either way SGTM

Reply:
👍
nice! I guess similarly to above, we can introduce the Clock interface later if it's needed (when there are multiple functions there to be mocked).